.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode/
```

Comment:
We should just use the `.gitignore` for Python projects that GitHub has -> https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
Resolved:
https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/pull/1/commits/5cd1182ba42cc63c3da4d924882b1128c7a5a151

Reply:
@mithro I'll not `resolve conversation`. I want you to approve the changes.

Reply:
LGTM.